contrib/gitian-descriptors/gitian-linux-parallel.yml

```diff
@@ -0,0 +1,141 @@
+---
+name: "zcash-5.3.0-rc1"
```

Comment:
Should this be "zcash-5.3.0"?

Reply:
I was going directly off what the current `gitian-linux.yml` file contained (which included an `rc1`)

Reply:
`make-release.py` has a function at https://github.com/zcash/zcash/blob/cc34f4f9bacdaaee64f5d80550229f001959efdb/zcutil/make-release.py#L377 to patch the gitian descriptor file, which should be changed to patch both files (equivalently, pass the filename into the function and call it twice).

Reply:
Oh right you already did that :-)